src/metabase/api/embed.clj

```diff
@@ -124,6 +124,7 @@
              qp          qp.card/process-query-for-card-default-qp}}]
   (let [unsigned-token (embed/unsign token)
         dashboard-id   (embed/get-in-unsigned-token-or-throw unsigned-token [:resource :dashboard])]
+    (events/publish-event! :event/dashboard-queried {:object-id dashboard-id :user-id api/*current-user-id*})
```

Comment:
It's a bit surprising that `:event/dashboard-queried` is published in the `process-query-for-dashcard-with-signed-token` function while `:event/card-read` is published in each endpoint handler separately. There's pros and cons either way and I don't have a strong opinion on it, but I think it should be consistent.